src/pages/Onboard.js

```diff
@@ -16,23 +17,51 @@ import { SingleSelect } from "../components/SingleSelect";
 import { CreatableMultiValueInput } from "../components/CreatableMultiValueInput";
 import { isEmailValid } from "../utils/email";
 
+import { getFlow, onboard } from "../services/flow";
+
 const Onboard = (props) => {
   const [state, setState] = useState({
     selectedFlow: null,
-    isLoading: false,
+    isLoading: true,
     onboardingEmails: null,
     error: null,
+    selectFlowOptions: [],
+    done: false,
   });
 
-  const { selectedFlow, onboardingEmails, error, isLoading } = state;
-
-  const selectFlowOptions = [
-    { label: "Software Intern", value: 1 },
-    { label: "Senior Backend Engineer", value: 2 },
-    { label: "Senior Frontend Engineer", value: 3 },
-    { label: "Junior Backend Engineer", value: 4 },
-    { label: "Junior Frontend Engineer", value: 5 },
-  ];
+  useEffect(() => {
+    getFlow()
+      .then((response) => {
+        let data = [];
+        response = response.data;
+        response.forEach((e) =>
```

Comment:
Same here change `response.forEach((e) =>` to `flowsArr.forEach((service) =>`. It would become more readable